### DISCUSSION TITLE
Media Manager: Upload vs. Replace

### DISCUSSION THREAD

**Discussion by technik-kvz** [joomla/joomla-cms#46992] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

If you upload a media file and the file name already exists, there is an error and upload stops. You have to manually delete the file before upload. If you upload a list of files (not only 1) this can consume much time.

### Describe the solution you'd like

A popup message to ask you for permission to replace the file, if already exists with an option "remember for all flies of this upload session" (if possible).

**Comment by Quy** [comment]
Closing as this is no longer the case.

![41225](https://github.com/joomla/joomla-cms/assets/368084/66645f44-7840-48a0-9254-abda14cf8afb)

**Comment by brianteeman** [comment]
please re-open. The dialog is only displayed if you try to drag'n'drop. If you try with the upload button then nothing happens

**Comment by Quy** [comment]
I got the dialog using the Upload button with Firefox and Chrome.

**Comment by brianteeman** [comment]
https://github.com/joomla/joomla-cms/assets/1296369/d5e47a5a-b406-4221-8819-d9a117454302

**Comment by Quy** [comment]
I can reproduce it per your video when uploading/reuploading in the same session, then no alert.

If you upload, refresh the page and reupload, then the alert is displayed.
If you upload to replace an existing image, then the alert is displayed.
